src/components/OrgSettings/General/OrgUpdate/OrgUpdate.tsx

```diff
@@ -238,108 +250,48 @@ function OrgUpdate(props: InterfaceOrgUpdateProps): JSX.Element {
-                autoComplete="off"
-                value={formState.address.line1}
-                onChange={(e) => handleInputChange('line1', e.target.value)}
-              />
-            </Col>
-            <Col sm={6} className="mb-1">
-              <Form.Control
-                placeholder={t('line2')}
-                autoComplete="off"
-                value={formState.address.line2}
-                onChange={(e) => handleInputChange('line2', e.target.value)}
-              />
-            </Col>
-          </Row>
-          <Row className="mb-1">
-            <Col sm={6} className="mb-1">
-              <Form.Control
-                placeholder={t('postalCode')}
-                autoComplete="off"
-                value={formState.address.postalCode}
-                onChange={(e) =>
-                  handleInputChange('postalCode', e.target.value)
-                }
-              />
-            </Col>
-            <Col sm={6} className="mb-1">
-              <Form.Control
-                placeholder={t('sortingCode')}
-                autoComplete="off"
-                value={formState.address.sortingCode}
-                onChange={(e) =>
-                  handleInputChange('sortingCode', e.target.value)
-                }
-              />
-            </Col>
-          </Row>
+
+          <Form.Label className={styles.orgUpdateFormLables}>
+            {tCommon('Location')}
+          </Form.Label>
+          <Form.Control
+            className={styles.textFields}
+            placeholder={tCommon('Enter Organization location')}
+            autoComplete="off"
+            required
+            value={formState.address.line1}
+            onChange={(e): void => {
+              handleInputChange('line1', e.target.value);
+            }}
+          />
+          <Form.Label htmlFor="orgphoto" className={styles.orgUpdateFormLables}>
+            {tCommon('displayImage')}:
+          </Form.Label>
+          <Form.Control
+            className={styles.customFileInput}
+            accept="image/*"
+            placeholder={tCommon('displayImage')}
+            name="photo"
+            type="file"
+            multiple={false}
+            onChange={async (e: React.ChangeEvent): Promise<void> => {
+              const target = e.target as HTMLInputElement;
+              const file = target.files && target.files[0];
+              if (file)
+                setFormState({
+                  ...formState,
+                  orgImage: await convertToBase64(file),
+                });
+            }}
+            data-testid="organisationImage"
+          />
           <Row>
-            <Col sm={6} className="d-flex mb-3">
-              <Form.Label className="me-3">
-                {t('userRegistrationRequired')}:
+            <Col sm={6} className="d-flex mb-4 mt-4 align-items-center">
+              <Form.Label className="me-3 mb-0 fw-normal text-black">
+                {t('Is Public')}:
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.